functions/Get-DbaDbUdf.ps1

```diff
@@ -97,22 +113,38 @@ function Get-DbaDbUdf {
 
             foreach ($db in $databases) {
 
-                $UserDefinedFunctions = $db.UserDefinedFunctions
+                $userDefinedFunctions = $db.UserDefinedFunctions
 
-                if (!$UserDefinedFunctions) {
+                if (!$userDefinedFunctions) {
                     Write-Message -Message "No User Defined Functions exist in the $db database on $instance" -Target $db -Level Verbose
                     continue
                 }
                 if (Test-Bound -ParameterName ExcludeSystemUdf) {
-                    $UserDefinedFunctions = $UserDefinedFunctions | Where-Object { $_.IsSystemObject -eq $false }
+                    $userDefinedFunctions = $userDefinedFunctions | Where-Object { $_.IsSystemObject -eq $false }
+                }
+
+                if ($Schema) {
+                    $userDefinedFunctions = $userDefinedFunctions | Where-Object { $_.Schema -in $Schema }
+                }
+
+                if ($ExcludeSchema) {
+                    $userDefinedFunctions = $userDefinedFunctions | Where-Object { $_.Schema -notin $ExcludeSchema }
```

Comment:
This is not a huge deal, but if we want to standardize, we should standardize against `Where-Object Schema -notin $ExcludeSchema`. Not only is it prettier, but it's faster too and recommended by Bruce Payette. Apparently, the `{}` brackets slow things down.

The only time brackets are required are when we need to compare more than one thing.

Reply:
You are rigth - just did a copy and paste. Will change that soon.